djangocms_versioning/admin.py

```diff
@@ -433,38 +432,29 @@ def edit_redirect_view(self, request, object_id):
             raise Http404
 
         # Redirect
-        # If the object is editable the cms editable view should be used, with the toolbar.
-        if is_editable_model(version.content.__class__):
-            url = get_object_edit_url(version.content)
-        # Or else, the standard edit view should be used
-        else:
-            url = reverse('admin:{app}_{model}_change'.format(
-                app=version.content._meta.app_label,
-                model=version.content._meta.model_name,
-            ), args=(version.content.pk,))
-        return redirect(url)
+        return redirect(get_editable_url(version))
```

Comment:
Perhaps change it so `get_editable_url` expects a content object? You're not using any other version property than `version.content` inside